strapi/strapi
# fix: stage change should not update modified status

### What does it do?
Changing the stage/assignee attribute made the document status go from publshed to modifed, confusing the users.

This PR ensures changhing those values will not change the publicaton status of the document, by keeping the same updated at value when making the update on the entry.


### Why is it needed?

Describe the issue you are solving.

### How to test it?

Go to the Content Manager
Go to a content type with a review workflow assigned (you will need a paid license)
Go to an entry with a published state
Change the stage of this entry
You will see the state not changed from Published to Modified


### Related issue(s)/PR(s)

Fixes https://github.com/strapi/strapi/issues/21429
